src/core/util/util.js

```diff
@@ -731,6 +731,16 @@ function UtilFactory($document, $timeout, $compile, $rootScope, $$mdAnimate, $in
       return current;
     },
 
+    /**
+     * Checks the sidenav's hide state
```

Comment:
@EladBezalel Done I'd really like to forget all that issues and create a new PR :smile: But in my newer PR's there aren't so much style improvements. Just learning the style you / the Team uses!